cep-releases.md

```diff
@@ -0,0 +1,95 @@
+<tr><td> Updated </td><td> July 20, 2022 </td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Describes a release cadence for all future `conda` versions starting with `22.7.0`.
+
+## Specification
+
+We propose regularly scheduled bimonthly (every two months) releases where the tagging/release occurs during the week (per ISO 8601, Monday through Thursday) of second Monday of the month.
+
+In a nod to our many different kinds of users we will propose a deprecation policy (to be defined in a later CEP) that allows for a slower adoption rate (i.e. users could update every 3-4 months instead).
+
+These objectives will be aided in the adoption of [CalVer](https://calver.org/):
+- `YY`: the major version will be the shortened year (22+)
+- `MM`: the minor version will be the shortened month (1-12)
+- `MICRO`: the micro/patch version is reset to zero every month and incremented for every additional release that occurs during that month (0+)
+
+This scheme will start with the July 2022 release of 22.7.0.
+
+This will result in the following regular release schedule:
+
+| Version | Release Week |
+|---|---|
+| 22.7.0 | 2022-07-11 |
+| 22.9.0 | 2022-09-12 |
+| 22.11.0 | 2022-11-14 |
+| 23.1.0 | 2023-01-09 |
+| 23.3.0 | 2023-03-13 |
+| 23.5.0 | 2023-05-08 |
+| 23.7.0 | 2023-07-10 |
+| 23.9.0 | 2023-09-11 |
+| ... | ... |
+
+> **Note**
+> Despite following a bimonthly release schedule we will permit releases to occur at any time between regular releases for hotfixes, security releases, or other high priority changes that require immediate release.
+
+To distinguish between the bimonthly release schedule and other optional releases the following release types are defined:
+
+- **regular release**: this is the regularly schedule bimonthly release
+- **optional release**: releases that may occur on the alternating months from regular releases
+- **hotfix release**: extra releases that may occur during any month to patch an earlier release of the same month (whether a regular or optional release)
+
+So it's entirely feasible to see the following releases:
+
+| Version | Release Type |
+|---|---|
+| 22.7.0 | regular |
+| 22.7.1 | hotfix |
+| 22.7.2 | hotfix |
+| 22.8.0 | optional |
+| 22.8.1 | hotfix |
+| 22.9.0 | regular |
+| 22.10.0 | optional |
+| 22.10.1 | hotfix |
+| ... | ... |
+
+## Motivation
+
+Remove ambiguity/maintainer guesswork of when and what warrants a release.
+
+## Backwards Compatibility
+
+Adopting this release schedule does not break any existing processes or schemes.
+
+We will entirely sidestep and avoid any confusion regarding years of rumored changes slated to be included in a `conda` 5 release since we won't even attain a 5 release with this new scheme. This is not to say that these desired changes wont occur, but rather that its unrealistic for all of these changes to be successfully (and safely) rolled out in a single release.
+
```

Comment:
What about breaking tools that require a certain version format? It seems like that won't be the case here, but it may be nice to briefly mention that too (i.e. that CalVer is compatible with SemVer).

Reply:
Added

Reply:
Added